Using sys_siglist is deprecated in favor of strsignal

Patch by R. Bosch

http://bugs.gentoo.org/103189

--- alsa-utils-1.0.9a/alsamixer/alsamixer.c
+++ alsa-utils-1.0.9a/alsamixer/alsamixer.c
@@ -2207,7 +2207,7 @@
 mixer_signal_handler (int signal)
 {
   if (signal != SIGSEGV)
-    mixer_abort (ERR_SIGNAL, sys_siglist[signal], 0);
+    mixer_abort (ERR_SIGNAL, strsignal(signal), 0);
   else
     {
       fprintf (stderr, "\nSegmentation fault.\n");
